first_models.tex

```diff
@@ -1,14 +1,164 @@
 \section{First Models}
 \label{sec:first_models}
 
-  \subsection{Bag of Words}
-  \label{sec:bag_of_words}
+\subsection{Bag of Words}
+\label{sec:bag_of_words}
+From the examination of the data in Section \ref{sec:word_frequency}, a bag-of-words classifier seems plausible. Through a direct comparison of the frequencies with which each author uses the words in a given sentence, a prediction may be formed.
 
-  \subsection{Forest}
-  \label{sec:forest}
+Let $\phi_w^a$ be the number of occurrences of some word $w$ in the corpus of sentences attributed to author $a$. The estimated probability of an author's usage of a word, $w^*$, is given by:
 
-  \subsection{DNN}
-  \label{sec:dnn}
+\begin{equation*}
```

Comment:
Do these equations need to be numbered or lettered?

Reply:
I don't think it serves any purpose and just clutters it.